### PR TITLE
Add Theming documentation

### PR DESCRIPTION
**Pull request by rurikoaraki**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [ ] win32 (Office)
- [ ] windows
- [ ] android

### Description of changes

Add some theming documentation to the docs pages folder. These pages are intended to be easy to approach and have basic examples on how to use our theming system

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [x] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

**Review by ksiler — APPROVED** [review]

File: docs/pages/Theming/DefaultThemes.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+```tsx
+<ThemeProvider theme={createAppleTheme()}>
+  <Button content={'Button'} />
+</ThemeProvider>
+```
+
+![Image of default button with default Apple theme on iOS applied in FURN](./assets/iOS.PNG)
+
+### Mac
```
Comment (Saadnajmi) [inline docs/pages/Theming/DefaultThemes.md:78]: nit, `macOS` to match `iOS` above.

File: docs/pages/Theming/DefaultThemes.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+### iOS
+
+```tsx
+<ThemeProvider theme={createAppleTheme()}>
+  <Button content={'Button'} />
+</ThemeProvider>
+```
+
+![Image of default button with default Apple theme on iOS applied in FURN](./assets/iOS.PNG)
```
Comment (Saadnajmi) [inline docs/pages/Theming/DefaultThemes.md:76]: Could we make all these files lowercase .png ? I think that's the more standard

Comment (rurikoaraki) [reply docs/pages/Theming/DefaultThemes.md:76]: Uhh sure let me try. Unfortunately the Windows OS doesn't count case changing as a rename so it kinda just keeps the case if I just try to rename

Comment (Saadnajmi) [reply docs/pages/Theming/DefaultThemes.md:76]: Ah. That's probably alright then. I forgot windows case matching is different.

Comment (rurikoaraki) [reply docs/pages/Theming/DefaultThemes.md]: Tried something, I think it worked

File: docs/pages/Theming/DefaultThemes.md (hunk 1 above)
Comment (rurikoaraki) [reply docs/pages/Theming/DefaultThemes.md]: Done, thanks!

**Review by Saadnajmi — APPROVED** [review]

**Pull request merged**